### PR TITLE
fix #70 stake out

### LINKED ISSUE #70: Notes on Stake out
Document the new feature about stake out:
https://github.com/lutraconsulting/input/pull/1845

### PR DESCRIPTION
**Pull request by alex-cit**
fix #70 
I need to redo the screenshots, so that they show the same features (something nicer than feature X) and reasonable GPS accuracy.

### CODE REVIEW

**Comment by PeterPetrik** [comment]
@tomasMizera can you look too please?

File: src/field/stake-out/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+:::
+
+
+1. Select a point in your map window. From the form select the **Stake out** icon:
+![stake out](./stake-out.png)
+
+2. Stake out panel will appear showing the current distance and a connecting line between your position and the point.
+![long range](./stake-out-long.png)
+
+3. Once you are closer to the point, the stake out panel will switch to short navigation mode.
```
Comment (tomasMizera) [inline src/field/stake-out/index.md:21]: Short mode is visible if user is less than 1m from destination point, but I am not sure if we want to mention that.

File: src/field/stake-out/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+![short range](./stake-out-short.png)
+
+4. Precise stake out of the point is highlighted by green:
```
Comment (tomasMizera) [inline src/field/stake-out/index.md:24]: It gets green when user is less than 10cm from target, but again, I am not sure if we want to mention that.

**Review by tomasMizera — CHANGES_REQUESTED** [review]
It looks great, thanks! I left two minor notes, I am not sure if we want to add precise distances or not. I will leave the decision with you guys.

I am only worried about one thing. I saw in the included pictures that the short navigation mode is not rendered properly (outer circle is cut on top and bottom):



@alex-cit If you are using landscape mode, can you please try it in portrait mode and take picture with nicely rendered circles? (note: https://github.com/lutraconsulting/input/issues/1948) If you are not able to get that, I can help you out with it, let me know :)

**Comment by alex-cit** [comment]
Screenshots will be updated later - https://github.com/MerginMaps/docs/issues/126

**Pull request merged**